.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "frontends/qt/external/SingleApplication"]
+	path = frontends/qt/external/SingleApplication
+	url = https://github.com/itay-grudev/SingleApplication.git
```

Comment:
why would i make it lower case instead of keeping the original name? what kind of inconsistency are you talking about?

Reply:
issues like this: https://dzone.com/articles/git-and-the-hell-of-case-sensitivity
just keep it simple, all lowercase, works best cross-platform. github doesn't care. this will work just as well:

```
[submodule "frontends/qt/external/singleapplication"]
	path = frontends/qt/external/singleapplication
	url = https://github.com/itay-grudev/singleapplication.git
```

Reply:
forked the repo and pointing to that now

Reply:
@x1ddos done lowecasing, though I really don't like changing original names. I doubt we will have a different file/folder with the same name but different case ever.